src/mixins/document-mixin.ts

```diff
@@ -17,6 +17,16 @@ export default class DocumentMixin extends Vue {
       validationErrorMsg: 'Document must be set to fit onto 8.5” x 11” letter-size paper.'
     }
   }
+  readonly DOCUMENT_TYPES = {
+    contInAuthorization: {
+      class: 'CORP',
+      type: 'CNTA'
```

Comment:
Can we have a more verbose type code?

Also, what about file uploads for Dissolutions (see CompleteAffidavit.vue) and Incorporations (see UploadMemorandum.vue and UploadRules.vue)?